include/tvm/runtime/data_type.h

```diff
@@ -231,7 +231,15 @@ inline int GetVectorBytes(DataType dtype) {
   int data_bits = dtype.bits() * dtype.lanes();
   // allow bool to exist
   if (dtype == DataType::Bool()) return 1;
-  return data_bits;
+  // allow int4/uint4/int1 to exist
+  if (dtype == DataType::Int(4) ||
+      dtype == DataType::UInt(4) ||
+      dtype == DataType::Int(1)) {
+    return data_bits;
```

Comment:
why date_bits here, instead of bytes? is there any trick here?

Reply:
Sorry, this is a mistake when merge code to latest code base, for int1/int4/uint4 1 is a more appropriate return value. I already fix it in my latest commit.